_plugins/html-beautify.rb

```diff
@@ -0,0 +1,33 @@
+require "jekyll"
+require "htmlbeautifier"
+
+module Jekyll
+  module Beautify
+    def self.init(site)
+      config = site.config
+      @include_paths = (config["html-beautify"] && config["html-beautify"]["include"]) || []
+    end
+
+    def self.include?(path)
+      return @include_paths.any? { |pattern| File.fnmatch(pattern, path) }
+    end
+
+    def self.process(output)
+      return HtmlBeautifier.beautify output
+    end
+  end
+end
+
+Jekyll::Hooks.register :site, :after_reset do |jekyll|
```

Comment:
You could pop these blocks inside the `Jekyll` module as well and lose the `Jekyl::` qualifiers. It'd make the code a wee bit cleaner in my opinion.

Reply:
Done.